### ISSUE TITLE
'stdout maxBuffer length exceeded' error when opening large JAR files

### ISSUE THREAD

**Issue by oxoxoxox** [recursean/JAR-Viewer-and-Decompiler-VSCode-Extension#6]
As https://github.com/microsoft/vscode-cpptools/issues/10107 mentioned,
we should modify the maxBuffer value, when opening large JAR files(such as 2MB JAR files).

Something like this:

```
const command = `java -jar ${cfrPath} --extraclasspath ${jarFilePath} ${filePath}`;
                cp.exec(command, {maxBuffer: 1024 * 5000}, async (error, stdout, stderr) => {
```

Recommand to allow user to set the maxBuffer value in the setting configuration.

**Comment by recursean** [comment]
Hey there! Thanks for trying the extension. 

This sounds like a good recommendation. I'll look into it more in the coming days and give you an update soon.

**Comment by recursean** [comment]
I just released version [1.2.1](https://marketplace.visualstudio.com/items?itemName=seanmcshane.jar-viewer-and-decompiler) of this extension that adds a new "cfrOutputSize" extension setting where you can specify the maximum number of kilobytes for the CFR output buffer. When the "stdout maxBuffer length exceeded" error is detected now, a new extension error message will display that contains a link to the new setting and recommends increasing the value. I was able to recreate the error myself and it seems this new setting helps resolve the error. 

I would love to hear your feedback on the new version and if this satisfies your request. I can address any other issues in future versions of the extension.

**Comment by oxoxoxox** [comment]
Good! It's working normally now
